index.html

```diff
@@ -71,128 +87,141 @@ <h1>Hey There. I’m Ronald.</h1>
     </main>
     <!-- works section -->
     <section class="works">
-      <h2 class="recent">My Recent Works</h2>
+      <div class="line">
+        <h2 class="recent">My Recent Works</h2>
+        <img src="./images/desky/Vector.svg" alt="line" />
+      </div>
 
       <div class="card">
         <div class="projimage">
           <img src="./images/projects/Img Placeholder.svg" alt="" />
         </div>
-        <h2 class="cardheader">Multi-Post Stories</h2>
-        <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
-        <ul class="languages">
-          <li>CSS</li>
-          <li>HTML</li>
-          <li>Bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-        <button>See project</button>
-      </div>
-      <div class="card2">
-        <div class="textitems">
-          <h2>Profesional Art Printing Data</h2>
+        <div class="text">
+          <h2 class="cardheader">Multi-Post Stories</h2>
           <p>
             A daily selection of privately personalized reads; no accounts or
-            sign-ups required. Has been the industry's standard.
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
           </p>
-          <ul>
+          <ul class="languages">
+            <li>CSS</li>
             <li>HTML</li>
             <li>Bootstrap</li>
             <li>Ruby</li>
           </ul>
+          <button>See project</button>
         </div>
-        <button>See project</button>
       </div>
-      <div class="card2">
-        <div class="textitems">
-          <h2>Profesional Art Printing Data</h2>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. Has been the industry's standard.
-          </p>
-          <ul>
-            <li>HTML</li>
-            <li>Bootstrap</li>
-            <li>Ruby</li>
-          </ul>
+      <div class="grids">
+        <div class="card2" id="card1">
+          <div class="textitems">
+            <h2>Data Dashboard Healthcare</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul>
+              <li>HTML</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button>See project</button>
```

Comment:
- [ ] As suggested by all the previous reviewers please change the background image of your first card to match with the figma design. I'll approve this project because I see you have utilized almost all of your extra reviews as well and this change is not that big however I would highly suggest you to implement the suggested change to make your design look snatched.


|Figma design |Your design|
|-----|-----|
|<img width="1228" alt="Screenshot 2023-06-10 at 8 14 30 PM" src="https://github.com/mj-e-boc/M-J-Portfolio/assets/44798044/2e70c74e-72a0-4872-9a7a-ef81b5296843">|<img width="1228" alt="Screenshot 2023-06-10 at 8 14 30 PM" src="https://github.com/mj-e-boc/M-J-Portfolio/assets/44798044/8e4ae8b1-7837-4155-a6d3-b75bf7128469">|